index.js

```diff
@@ -65,6 +72,95 @@ for( var i = 0; i < keys.length; i++ ) {
 	else {
 		input.innerHTML += btnVal;
 	}
-		event.preventDefault();
+	event.preventDefault();
+	}
```

Comment:
Does this bracket close the function declared on line 7, if so the bracket should be outdented in order to be correctly aligned.
Also, the `event.PreventDefault()` should be declared at the top of the function